### PR TITLE
Fixed naming to make it more appropriate

### PR DESCRIPTION
**Pull request by WickedSik**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,4 +44,9 @@ Thumbs.db
 *.jar
 *.rar
 *.tar
-*.zip
\ No newline at end of file
+*.zip
+
+# Personal files
```
Comment (Rothiss) [inline .gitignore]: These should either be in your global gitignore, remove from stage or these files shouldn't be in the folder you checked out. Or maybee we should just add a scripts/custom/private folder in which we could place these kind of files.

Comment (WickedSik) [reply .gitignore]: I'm thinking that global gitignore should be the way to go. I currently do not have one.

**Comment by WickedSik** [comment]
- Added nginx fix
- Removed personal ignores

**Comment by Rothiss** [comment]
Your rebuild and up scripts are in this, should be in here. Remember the global git ignore?

**Comment by WickedSik** [comment]
#insert-curse-word# . You're right, I'll get it done.

**Comment by Rothiss** [comment]
See: https://github.com/Spring-Digital/sprdock/pull/65/files

**Pull request closed without merging**